third_party/move/move-compiler/src/command_line/compiler.rs

```diff
@@ -95,6 +97,7 @@ impl<'a> Compiler<'a> {
     pub fn from_package_paths<Paths: Into<Symbol>, NamedAddress: Into<Symbol>>(
         targets: Vec<PackagePaths<Paths, NamedAddress>>,
         deps: Vec<PackagePaths<Paths, NamedAddress>>,
+        known_attributes: &BTreeSet<String>,
```

Comment:
Do we really need to change this API here? I don't see a use case where known_attributes would be passed into the compiler like this.

Reply:
The problem is that there are a huge number of paths into the compiler today.  Maybe they shouldn't all be there, but I've traced them all (see https://pastebin.com/TH1nwNXs for some of my notes).